plugins/puppetdb/README.md

```diff
@@ -0,0 +1,44 @@
+# PuppetDB Plugin
+
+A frontend plugin to integrate PuppetDB with Backstage. When combined with the
+[catalog-backend-module-puppetdb](https://github.com/backstage/backstage/blob/master/plugins/catalog-backend-module-puppetdb/README.md) plugin, this
+frontend plugin allows viewing PuppetDB reports, including their logs and events, of Backstage resource entities.
+
+![Reports list](https://raw.githubusercontent.com/backstage/backstage/42b65232e763d3e39e2e641b105d2ad469db7a59/plugins/puppetdb/assets/Reports.png)
```

Comment:
The general practice is to have a `/docs/images` folder and put these there, feel free to follow that. You also have some other example in the PR description that would be great to see here. These screenshots really help give people an idea of the features and are super helpful.

Reply:
Actually looks like you have them in the `/assets` folder, that should work fine too. Would still be great to see them in the README